src/server/object_services/object_server.js

```diff
@@ -155,6 +155,27 @@ async function put_object_tagging(req) {
     };
 }
 
+/**
+ *
+ * put_object_property
+ *
+ */
+async function put_object_property(req) {
+    throw_if_maintenance(req);
+    load_bucket(req);
+    const obj = await find_object_md(req);
+    const info = get_object_info(obj);
+
+    await MDStore.instance().update_object_by_id(
+        obj._id, req.rpc_params.update, undefined, undefined
```

Comment:
I suggested removing this function, but in general, you don't need to explicitly pass `undefined`

Reply:
👍

Just note the style of MDStore.instance().update_object_by_id() references in object_server.js

[line 149-151](https://github.com/noobaa/noobaa-core/blob/cc184f7fbff7690bf8f31062b60d4d370b62df0f/src/server/object_services/object_server.js#L149-L151)
```js
    await MDStore.instance().update_object_by_id(
        obj._id, { tagging: req.rpc_params.tagging }, undefined, undefined
    );
```

[line 190-192](https://github.com/noobaa/noobaa-core/blob/cc184f7fbff7690bf8f31062b60d4d370b62df0f/src/server/object_services/object_server.js#L190-L192)
```js
    await MDStore.instance().update_object_by_id(
        obj._id, undefined, { 'tagging': 1 }, undefined
    );
```

[line 275-284](https://github.com/noobaa/noobaa-core/blob/cc184f7fbff7690bf8f31062b60d4d370b62df0f/src/server/object_services/object_server.js#L275-284)
```js
    await MDStore.instance().update_object_by_id(
        obj._id, {
            lock_settings: {
                legal_hold: {
                    status: req.params.legal_hold.status
                },
                retention,
            }
        }, undefined, undefined
    );
```

Those are the first three references and the list goes on...

Reply:
that is just a general comment, which is probably relevant to other places in the code :) 
in any case, this function should be removed, and there is no need to fix the code